common/traits/wc_general_traits.txt

```diff
@@ -21,4 +21,156 @@
 	can_have_children = no
 	disables_combat_leadership = yes
 	shown_in_ruler_designer = no
-}
\ No newline at end of file
+}
+
+#Tauren Event Traits
+#Earth Mother Blessings
+wc_tauren_EM_wolf = {
+	category = lifestyle
+	prowess = 5
+	intrigue = 2
+	health = 0.25
+	attraction_opinion = 20
+	monthly_prestige_gain_mult = 0.2
+	hostile_scheme_resistance_mult = 0.3
+
+	ai_boldness = medium_positive_ai_value
+	ai_honor = low_positive_ai_value
+	ai_sociability = low_positive_ai_value
+	ai_amenity_target_baseline = 0.2
+	ai_amenity_spending = 0.1
+	
+	shown_in_ruler_designer = no
+	icon = "forest_fighter.dds"
```

Comment:
Make new traits into 2 groups
`group = wc_tauren_EM` for positive ones
and
`group = wc_tauren_EM_NEG` for negative ones
So you can use `has_trait = wc_tauren_EM` or `has_trait = wc_tauren_EM_NEG` it checks all traits from this groups. In case you add new one. And its better to use instead of big `OR ={ has_trait... has_trait =... has_trait =... }`. So replace those checks in events with this group check

Reply:
I have already asked the 2D designers to make new icons for the traits, but they have yet to finish them, right now I picked workable placeholder traits till they complete their work. I'm not sure whether they have started yet or not though so should I wait till they finish it whenever that is or should we push this event pack when it's finished and then make a small update when the icons are ready.

Reply:
we dont plan to release any patch in steam soon, so we can wait till they finish icons. Even if they wont be able to finish, in worst case i can do it myself like in 30 min before next steam release